app/models/guid.ts

```diff
@@ -1,11 +1,20 @@
+import { AsyncBelongsTo, belongsTo } from '@ember-data/model';
 import { computed } from '@ember/object';
 import { singularize } from 'ember-inflector';
+import CustomFileMetadataRecordModel from 'ember-osf-web/models/custom-file-metadata-record';
+import CustomItemMetadataRecordModel from 'ember-osf-web/models/custom-item-metadata-record';
 
 import OsfModel from './osf-model';
 
 export type ReferentModelName = 'file' | 'node' | 'preprint' | 'registration' | 'user';
 
 export default class GuidModel extends OsfModel {
+    @belongsTo('custom-metadata', {inverse: 'guid', polymorphic: true})
```

Comment:
hm, does this expect a `custom-metadata` parent model to exist? might be good to add that (and perhaps put the `guid` relationship there)

Reply:
Interesting that nothing broke from that.

Reply:
yeah i wonder if it waits to break until it has an actual record to break on...